src/templates/base.html

```diff
@@ -119,7 +126,6 @@
         {% endfor %}
       {% endif %}
       {% block content %}
-        <p>Coming soon!</p>
```

Comment:
It doesn't seem necessary to remove this line and leaving it in doesn't seem to cause any issue with the forum page now either.